web/schemas/ID1A3.json

```diff
@@ -286,14 +444,56 @@
         "placeholder": "80.725"
     },
     {
-        "key": "BeamlineSetupDocument",
-        "type": "string",
+        "key": "AttenMaterial",
+        "type": "list_str",
         "optional": true,
-        "multiple": false,
+        "multiple": true,
         "section": "Beam",
-        "description": "Beamline setup document location",
-        "placeholder": "/"
+        "description": "List of attenuator materials used",
+        "placeholder": "steel, aluminum",
+        "value": [
+            "",
+            "Steel",
+            "Aluminum"
+        ]
     },
+    {
+        "key": "AttenThickness",
+        "type": "float64",
+        "optional": true,
+        "multiple": true,
+        "section": "Beam",
+        "description": "Thickness [mm] of attenuator(s) - enter one value for each attenutor material",
+        "placeholder": "0.25, 3.0"
```

Comment:
This is wrong. You should either define single number for placeholder or if you expect a list of numbers you should change the type. Please clarify.

Reply:
Fixed. A similar fix was applied to ID3A.json also. Made a few more minor edits to both ID1A3.json and ID3A.json.